lib/src/generators/openapi3/__spec-examples__/contract-with-schemaprops.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  api,
+  body,
+  endpoint,
+  headers,
+  request,
+  response,
+  String,
+  Integer
+} from "@airtasker/spot";
+
+@api({ name: "contract" })
+class Contract {}
+
+@endpoint({
+  method: "GET",
+  path: "/users"
+})
+class EndpointWithExampleOnHeaders {
+  @request
+  request(
+    @headers
+    headers: {
+      /** property-schemaprop description for string
+       * @schemaprop minLength
+       * 12
+       * @schemaprop maxLength
+       * 20
+       * @schemaprop pattern
```

Comment:
It may be a bit more effort to implement, but what do you think of using separate tags for each schema prop? i.e.

```ts
/**
  * @oaSchemaMinLength 12
  * @oaSchemaMaxLength 20
  * @oaSchemaPattern "^[0-9a-z_]+$"
  */
```

Reply:
Hi @lfportal ,

This change is a huge refactor, so could we do like `@oa3serverVariables` for generic name like you sugested on the other comment?
So to rename it like `@oaschemaProp`

What do you think?

Reply:
@Thodin3 makes sense, this approach is more practical for now. A larger refactor can be considered later. 👍

Reply:
ok